CMakeLists.txt

```diff
@@ -109,9 +109,11 @@ pkg_check_modules(deps REQUIRED IMPORTED_TARGET
     wayland-server wayland-client wayland-cursor wayland-protocols
     cairo pango pangocairo pixman-1
     libdrm libinput hwdata libseat libdisplay-info libliftoff libudev gbm
-    hyprwayland-scanner>=0.3.6 hyprlang>=0.3.2 hyprcursor>=0.1.7
+    hyprlang>=0.3.2 hyprcursor>=0.1.7
 )
 
+find_package(hyprwayland-scanner 0.3.6 REQUIRED)
```

Comment:
@vaxerski do we need 0.3.7?

Reply:
now we do anyways (see conflict)

Reply:
Right. @cnt0 can you update it?

Reply:
fixed & rebased

github seems to show false diffs in this PR, I have no idea why. `CMakeLists.txt` is the only file here that is actually changed compared to the main branch.

Reply:
Caused by merging main into your branch. A rebase should fix it. If it doesn't, `git rebase --interactive HEAD~10` and `delete` any commits that aren't your own.